pretext/ExtraTopics/SearchSortAlgorithms.ptx

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<section xml:id="extratopics-algorithms">
+  <title>Search and Sort Algorithms</title>
+
+  <p>In computer science, we like algorithms! There are two types of algorithms
```

Comment:
computer science is the study of algorithms.

Reply:
I still think we should say Computer Science is the study of algorithms... rather than we like algorithms.